app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SubmissionsController < ApplicationController
+  def new
+    @submission = Submission.new
+  end
+
+  def create
+    @submission = Submission.create(new_submission_params)
+
+    respond_with @submission
+  end
+
+  def show
+    @submission = Submission.includes(:poems).find(params[:id]) ||
+      Submission.new
+  end
+
+  private
+
+  def submission_params
+    params.require(:submission).
+      permit(:title, :submitted_to, :status, poem_ids: [])
```

Comment:
this is nitpicky but personally I would format this as

```
params.require(:submission).permit(
  :poem_ids: [],
  :status,
  :submitted_to,
  :title,
)
```

It makes the most interesting part (the params this form accepts) more clear.